### ISSUE TITLE
Add missing and fix alt-texts

### ISSUE THREAD

**Issue by simomps** [digitalfabrik/entitlementcard#2729]
### Describe the Problem

There are items without any alt-text. 

- The buttons on the map are without alt-text (center map on current position & information button)
- On IOS the icons next to the search results gives out the file path as an alt-text
- In the intro slides the headings are not declared as such 
- Intro slides: the images have no alt-text 
- Intro slides: the points at the bottom of the page have no alt text (related issue: https://github.com/digitalfabrik/entitlementcard/issues/2728 )

### Describe Your Preferred Solution

Add missing alt-text

### Describe Possible Alternatives

No other suggestion

### Related Issues

https://github.com/digitalfabrik/entitlementcard/issues/2716
https://github.com/digitalfabrik/entitlementcard/issues/2728 Intro-slide page points

### Additional Information

<img width="1920" height="1080" alt="Image" src="https://github.com/user-attachments/assets/036166d7-6395-4c0b-a1fb-7858392de9bc" />
<img width="1920" height="1080" alt="Image" src="https://github.com/user-attachments/assets/836b2b07-c53a-4634-8395-3d7a9dc7476e" />
<img width="1920" height="1080" alt="Image" src="https://github.com/user-attachments/assets/f29deed4-3e8c-4bb8-b7f2-6ab3e2f1e77b" />
<img width="1920" height="1080" alt="Image" src="https://github.com/user-attachments/assets/ae733b62-f787-49f0-9589-a1b394b172ea" />
